src/main/java/common/helpers/BrowserHelper.java

```diff
@@ -14,6 +14,8 @@
 
 public class BrowserHelper {
 
+    private static final BrowserHelper instance = new BrowserHelper();
```

Comment:
No need to create new instance. This is a helper class, so make all the necessary methods to public static modifier.